docs/services/frame-system/_index.md

```diff
@@ -288,127 +265,13 @@ The resulting tree of reference frames looks like:
 
 ## Access the frame system
 
-The [Robot API](https://github.com/viamrobotics/api/blob/main/proto/viam/robot/v1/robot.proto) supplies the following method to interact with the frame system:
+The [Machine Management API](/appendix/apis/robot/) supplies the following methods to interact with the frame system:
 
 <!-- prettier-ignore -->
 | Method Name | Description |
 | ----- | ----------- |
```

Comment:
removed this in favor of linking to the methods (which we now have both documented) bc I think that makes more sense/leaves us not having to duplicate, lmk and I can put them back

Reply:
nice